src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -312,6 +328,33 @@ export const useBalanceChartData: UseBalanceChartData = args => {
   const balances = useAppSelector(state =>
     selectPortfolioCryptoBalancesByAccountId(state, accountId)
   )
+
+  // Get total delegation
+  // TODO(ryankk): consolidate accountSpecifiers creation to be the same everywhere
+  const cosmosCaip2: CAIP2 = caip2.toCAIP2({
+    chain: ChainTypes.Cosmos,
+    network: NetworkTypes.COSMOSHUB_MAINNET
+  })
+
+  const accountSpecifiers = useSelector(selectAccountSpecifiers)
+  const account = accountSpecifiers.reduce((acc, accountSpecifier) => {
+    if (accountSpecifier[cosmosCaip2]) {
+      acc = accountSpecifier[cosmosCaip2]
+    }
+    return acc
+  }, '')
+
+  // TODO(ryankk): this needs to be removed once staking data is keyed by accountSpecifier instead of caip10
```

Comment:
As we've discussed yesterday that accountSpecifiers will probably need to be revamped in the future to be the same everywhere (we use non-consistent terminology to define an "account specifier" and that's confusing), could we use this new `selectPubkeyishByChainId` selector for now? https://github.com/shapeshift/web/pull/1357/files#diff-bcccd934c3fc099ae907180ecdaa60f38a99865289682fc66896ddba29b6e817R14

Then l.334-348 can go in favor of selecting a la:
```
  const accountSpecifiersForChainId = useAppSelector(state =>
    selectPubkeyishByChainId(state, asset?.caip2)
  )
  const accountSpecifier = accountSpecifiersForChainId?.[0]
```

FYI @0xdef1cafe ^

Reply:
Im down to make this change after the other PR is merged but don't want to hold up this PR waiting for it

Reply:
wtf is `pubkeyish`?

Reply:
I agree "pubkeyish" needs more documentation about why its needed and how its used with the selectors.

One issue I have with the new staking selectors is they are named ending in "AccountSpecifier" but its a not the same "AccountSpecifier" we use other places. Can be very confusing and might be worth renaming

Reply:
agree that the terminology is obtuse and i'll take the blame for it.

what we're doing with `eip155:1:0xdef1cafe` is not strictly caip10, as it doesn't work for bitcoin, as we're calling unchained with x/y/zpub

they're not strictly pubkeys either

`pubkeyish` is something @gomesalexandre and i spitballed as better than `AccountSpecifier` but that's clearly not popular either

we should probably refer to the x/y/zpub or eth/cosmos account as an `AccountSpecifier`, and the full `eip155:1:0x....` type strings as `AccountId`s

Reply:
> Im down to make this change after the other PR is merged but don't want to hold up this PR waiting for it

it's like moving from one file to another?